### PR TITLE
refactor: use simple items in basic layouts examples

### PR DESCRIPTION
**Pull request by sissbruecker**
Before:
Bildschirmfoto 2024-12-13 um 12 29 35

After:
Bildschirmfoto 2024-12-13 um 12 29 48

Closes https://github.com/vaadin/docs/issues/3995

### CODE REVIEW

File: frontend/demo/component/basiclayouts/basic-layouts-spacing-variants.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import 'Frontend/demo/init'; // hidden-source-line
-import '@vaadin/button';
-import '@vaadin/radio-group';
 import '@vaadin/vertical-layout';
+import '@vaadin/radio-group';
```
Comment (web-padawan) [inline frontend/demo/component/basiclayouts/basic-layouts-spacing-variants.ts:3]: nit: we generally keep imports sorted alphabetically so this change seems unnecessary.

File: src/main/java/com/vaadin/demo/component/basiclayouts/BasicLayoutsHorizontalLayout.java [hunk 2]
```diff
@@ -12,13 +11,20 @@ public BasicLayoutsHorizontalLayout() {
...
+        Div item2 = new Div("Item 2");
+        item2.setClassName("example-item");
+        
+        Div item3 = new Div("Item 3");
+        item3.setClassName("example-item");
+        
+        layout.add(item1, item2, item3);
         // end::snippet[]
-
+
```
Comment (web-padawan) [inline src/main/java/com/vaadin/demo/component/basiclayouts/BasicLayoutsHorizontalLayout.java:26]: nit: trailing whitespace.

**Review by web-padawan — APPROVED** [review]

File: frontend/demo/component/basiclayouts/basic-layouts-spacing-variants.ts (hunk 1 above)
Comment (sissbruecker) [reply frontend/demo/component/basiclayouts/basic-layouts-spacing-variants.ts:2]: Done

File: src/main/java/com/vaadin/demo/component/basiclayouts/BasicLayoutsHorizontalLayout.java (hunk 2 above)
Comment (sissbruecker) [reply src/main/java/com/vaadin/demo/component/basiclayouts/BasicLayoutsHorizontalLayout.java]: Done

**Review by DiegoCardoso — APPROVED** [review]

**Pull request merged**